.github/workflows/release-notes-issue.yml

```diff
@@ -0,0 +1,106 @@
+# release-notes-issue.yml
+# Updates a release notes tracking issue with the generated release notes.
+# The issue to update is set in the RELEASE_NOTES_ISSUE secret.
+
+name: "Update release notes issue"
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - ".changeset/**"
+
+  # Allow manually triggering this workflow from the web UI
+  workflow_dispatch:
+    inputs:
+      issueNumber:
+        description: 'Issue number to update'
+        default: 22101
```

Comment:
Is this the RELEASE_NOTES_ISSUE? Should this be using the variable as well?

Reply:
Can't use a variable for a default value. I just removed the config option. If we want to change the issue num, we can change the variable itself.